corehq/tabs/tabclasses.py

```diff
@@ -2442,6 +2442,40 @@ def _is_viewable(self):
                 and not is_request_using_sso(self._request))
 
 
+class AttendanceTrackingTab(UITab):
```

Comment:
We must just confirm if we want the AT to be accessed through here. I can't remember who mentioned that we might not want to do this?

Reply:
I confirmed with Mary that we're going this way.